README.md

```diff
@@ -81,7 +81,8 @@ Jawn also supports asynchronous parsing, which allows users to feed
 the parser with data as it is available. There are three modes:
 
 * `SingleValue` waits to return a single `J` value once parsing is done.
-* `UnwrapArray` if the top-level element is an array, return values as they become available.
+* `UnwrapArray` if the top-level element is an array, return values as they become available. Set `multiValue` to `true`
+if you want to support multiple top level array's.
```

Comment:
s/array's/arrays/

Reply:
I have rebased the commit with this change and force pushed, thanks for the pickup.